concepticondata/conceptlists/Nagano-2013-1256.tsv

```diff
@@ -0,0 +1,1257 @@
+Nagano-2013-1256-803	803	1434	beast	958	WILD ANIMAL	
+Nagano-2013-1256-804	804	1435	pack animal			
+Nagano-2013-1256-805	805	1436	livestock	631	LIVESTOCK	
+Nagano-2013-1256-806	806	1438	bird	937	BIRD	
+Nagano-2013-1256-807	807	1440	fish	227	FISH	
+Nagano-2013-1256-808	808	1442	worm	1219	WORM	
+Nagano-2013-1256-809	809	1444	dog	2009	DOG	
+Nagano-2013-1256-810	810	1446	cat	1208	CAT	
+Nagano-2013-1256-811	811	1447	lynx			
+Nagano-2013-1256-812	812	1448	wildcat	2241	WILD CAT	
+Nagano-2013-1256-813	813	1450	horse	615	HORSE	
+Nagano-2013-1256-814	814	1451	donkey	1862	DONKEY	
+Nagano-2013-1256-815	815	1452	saddle	3065	SADDLE	
+Nagano-2013-1256-816	816	1453	mule	890	MULE	
+Nagano-2013-1256-817	817	1455	cattle	1153	CATTLE	
+Nagano-2013-1256-818	818	1456	cattle	1153	CATTLE	
+Nagano-2013-1256-819	819	1457	yak	2477	YAK	
+Nagano-2013-1256-820	820	1458	female yak	2479	FEMALE YAK	
+Nagano-2013-1256-821	821	1459	wild yak			
+Nagano-2013-1256-822	822	1460	dzo (ox x female yak)			
+Nagano-2013-1256-823	823	1462	sheep	1331	SHEEP	
+Nagano-2013-1256-824	824	1463	bharal, blue sheep			
+Nagano-2013-1256-825	825	1464	argali sheep			
+Nagano-2013-1256-826	826	1466	goat	1502	GOAT	
+Nagano-2013-1256-827	827	1468	pig	1337	PIG	
+Nagano-2013-1256-828	828	1470	hen, cock	1318	CHICKEN	
+Nagano-2013-1256-829	829	1472	tiger	846	TIGER	
+Nagano-2013-1256-830	830	1474	bear	1820	BEAR	
+Nagano-2013-1256-831	831	1476	wolf	522	WOLF	
+Nagano-2013-1256-832	832	1478	fox	1312	FOX	
+Nagano-2013-1256-833	833	1480	deer	1936	DEER	
+Nagano-2013-1256-834	834	1482	ape			
+Nagano-2013-1256-835	835	1483	monkey	1350	MONKEY	
+Nagano-2013-1256-836	836	1485	rabbit	1136	RABBIT	
+Nagano-2013-1256-837	837	1487	mouse	1491	MOUSE	
+Nagano-2013-1256-838	838	1489	elephant	1290	ELEPHANT	
+Nagano-2013-1256-839	839	1491	camel	1871	CAMEL	
+Nagano-2013-1256-840	840	1493	bat	1793	BAT	
+Nagano-2013-1256-841	841	1495	eagle	1905	EAGLE	
+Nagano-2013-1256-842	842	1497	crane	1830	CRANE	
+Nagano-2013-1256-843	843	1499	swallow	3077	SWALLOW (BIRD)	
+Nagano-2013-1256-844	844	1500	little cuckoo	2934	CUCKOO	
+Nagano-2013-1256-845	845	1502	pigeon	1853	DOVE	
+Nagano-2013-1256-846	846	1503	pigeon	1853	DOVE	
+Nagano-2013-1256-847	847	1505	crow	913	CROW	
+Nagano-2013-1256-848	848	1506	crow	913	CROW	
+Nagano-2013-1256-849	849	1507	owl	735	OWL	
+Nagano-2013-1256-850	850	1509	sparrow	1854	SPARROW	
+Nagano-2013-1256-851	851	1511	butterfly	1791	BUTTERFLY	
+Nagano-2013-1256-852	852	1513	ant	587	ANT	
+Nagano-2013-1256-853	853	1515	spider	843	SPIDER	
+Nagano-2013-1256-854	854	1517	honey-bee	665	BEE	
+Nagano-2013-1256-855	855	1519	mosquito	1509	MOSQUITO	
+Nagano-2013-1256-856	856	1521	fly			
+Nagano-2013-1256-857	857	1523	flea	232	FLEA	
+Nagano-2013-1256-858	858	1525	louse	1392	LOUSE	
+Nagano-2013-1256-859	859	1527	snake	730	SNAKE	
+Nagano-2013-1256-860	860	1529	lizard	632	LIZARD	
+Nagano-2013-1256-861	861	1530	frog	503	FROG	
+Nagano-2013-1256-862	862	1532	horn	1393	HORN (ANATOMY)	
+Nagano-2013-1256-863	863	1534	hair			
```

Comment:
1040 HAIR

Reply:
This is actually animal hair, which the languages distinguish from both human hair and fur. In concepticon we have only FUR and HAIR OR FUR, but no ANIMAL HAIR (HAIR (BODY) 189 is defined as human hair).